src/BlazorUI/Bit.BlazorUI.Extras/Styles/extra-components.scss

```
@@ -4,6 +4,7 @@
 @import "../Components/ErrorBoundary/BitErrorBoundary.scss";
 @import "../Components/Flag/BitFlag.scss";
 @import "../Components/InfiniteScrolling/BitInfiniteScrolling.scss";
+@import "../Components/Map/BitMap.scss";
```

Stylelint (17.11.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)